xrpl/models/utils.py

```diff
@@ -1,10 +1,73 @@
 """Helper util functions for the models module."""
 
+import re
 from dataclasses import dataclass, is_dataclass
-from typing import Any, Dict, List, Type, TypeVar, cast
+from typing import Any, Dict, List, Optional, Pattern, Type, TypeVar, cast
+
+from typing_extensions import Final
 
 from xrpl.models.exceptions import XRPLModelException
 
+HEX_REGEX: Final[Pattern[str]] = re.compile("[a-fA-F0-9]*")
+
+# Credentials are represented in hex. Whilst they are allowed a maximum length of 64
+# bytes, every byte requires 2 hex characters for representation
+_MAX_CREDENTIAL_LENGTH: Final[int] = 64 * 2
```

Comment:
I would just set it to 128 since its comment explains the calculated value.
```suggestion
_MAX_CREDENTIAL_LENGTH: Final[int] = 128
```

Reply:
@khancode  I prefer the existing version, it provides an explanation for the 128 required bytes